### ISSUE TITLE
Add support for aliases

### ISSUE THREAD

**Issue by MarcinIN** [inausoft/netCLI#17]
This is common to allow for shortcuts in options for example to allow `-f` instead of `--force`.
It should result is something like:

```cs
[Command("move", "Moves files between locations.")]
    public class MoveCommand
    {
        [Option("force", "Indicates weather or not files should be overwritten.", IsOptional = true, Alias = "f")]
        public bool IsForce { get; set; }

        [Option("from", "Current files locations.")]
        public string PathFrom { get; set; }

        [Option("to", "Destination path for the files.")]
        public bool PathTo { get; set; }
    }
```

**Comment by MarcinIN** [comment]
@matiduck @jmarlew any insight? What about:
```cs
[Option("force|f", "Indicates weather or not files should be overwritten.", IsOptional = true)]
public bool IsForce { get; set; }
```

**Comment by jmarlew** [comment]
I like the `Option("force|f", ...` way.

Maybe it's not related strictly to this CR but I thought - what if I create a command class that defines options with duplicate names (or aliases)? Eg. `force|f` and `foo|f`?

**Comment by MarcinIN** [comment]
A `InvalidOperationExcepton` during `Map` invocation - assume. It should have been this way already cause you might try to Map 2 different commands with the same name - but is it ...?

**Comment by MarcinIN** [comment]
> It should have been this way already cause you might try to Map 2 different commands with the same name - but is it ...?

Well it was not - but I've adjusted it in the #30.

**Comment by matiduck** [comment]
Agreed that allowing short argument should be included, but I'm against using word `Alias` when we refer to those options. `Short` maybe would be better?

**Comment by MarcinIN** [comment]
Agreed that `alias` refers to https://en.wikipedia.org/wiki/Alias_(command) which is something else. What about `Synonym` - I'm hesitant in regards to `Short`. What do you think: @jmarlew @xHoody

**Comment by MarcinIN** [comment]
Merged to 2.1
